source/features/clean-dashboard.css

```diff
@@ -49,11 +49,6 @@
 	margin-bottom: 0;
 }
 
-/* Hide footer date on Created Repository events */
-.rgh-clean-dashboard .dashboard .repo .f6:is(.color-text-secondary, .color-fg-muted) {
-	display: none;
-}
```

Comment:
I'm not sure but this footer date is the last activity date of the repo it will differ from the repo create date. Normally this is not a issue since people seldom read past events, but hiding it altogether is inconsistent with other cards.